### PR TITLE
Finish Implementation

### PR DESCRIPTION
**Pull request by TheTripleV**
Making this PR to get comments on `sphinxext/rediraffe.py`.

TODO:
- [x] Finish implementation
- [x] Write tests
- [x] Write readme
- [x] CI

### CODE REVIEW

**Comment by TheTripleV** [comment]
pypy3 on windows can't build a testing dependency, cryptography, because openssl isn't available. Either pypy3 on windows needs to be removed as a testing platform or openssl must be installed on windows.

**Comment by Daltz333** [comment]
I think it's fine to remove the pypy3 on Windows (or in general) as long as we can guaranteed stability for regular Python 3 on Windows.

**Comment by TheTripleV** [comment]
Warning added.

File: setup.py [hunk 1]
```diff
@@ -11,14 +11,14 @@
     long_description = readme.read()
 
 setuptools.setup(
-    name="sphinxext-linkcheckdiff",
+    name="sphinxext-rediraffe",
     version=version,
     author="Vasista Vovveti",
     author_email="vasistavovveti@gmail.com",
     description="Sphinx Extension to run diff-only linkchecks",
```
Comment (Daltz333) [inline setup.py:18]:
```suggestion
    description="Sphinx Extension that redirects non-existent pages to working pages.",
```

**Pull request merged**